### ISSUE TITLE
Can vector-compat supports SVG rendering animation path？

### ISSUE THREAD

**Issue by zhe8300975** [wnafee/vector-compat#31]
Can vector-compat supports SVG rendering animation path？  For example “trimPathStart”。I find vector-compat have "vc_trimPathStart"   but I would not use it。 

if vector-compat supports SVG rendering animation path，How should I use。

I write the Animator， can not be used

<?xml version="1.0" encoding="utf-8"?>
<objectAnimator xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:duration="1000"
    android:propertyName="vc_trimPathStart"
    android:valueFrom="0"
    android:valueTo="1"
    android:valueType="pathType"
    app:vc_valueType="pathType">
    </objectAnimator>

**Comment by twyatt** [comment]
I'm not certain of vector-compat's support for `trimPathStart`, but based on Android's [VectorDrawable](http://developer.android.com/reference/android/graphics/drawable/VectorDrawable.html), it looks as though trimPathStart is a `float` (not `pathType`).

**Comment by zhe8300975** [comment]
thank you ，
When parsing collapse, so I tried pathType

`
 if (valueType == VALUE_TYPE_PATH) {
            evaluator = setupAnimatorForPath(anim, arrayAnimator);
        } else {
            throw new IllegalArgumentException("target is not a pathType target");
        }
`

**Comment by Emasoft** [comment]
Yes, supporting SVG animations would be awesome! Now you can make them with Adobe Animate CC and exporting them with flash2svg. Look at those svg animated emoticons in the Viber app: 
http://www.tbyrne.org/viber-animated-svgs
